src/videocontext.js

```diff
@@ -80,10 +84,20 @@ export default class VideoContext {
             return;
         }
 
+        this._webAudioEnabled = webAudioEnabled;
+        if (this._webAudioEnabled) {
+            this._audioCtx = new AudioContext({
```

Comment:
We should add a `webkitAudioContext` fallback for Safari https://developer.apple.com/library/archive/documentation/AudioVideo/Conceptual/Using_HTML5_Audio_Video/PlayingandSynthesizingSounds/PlayingandSynthesizingSounds.html

Reply:
`this._audioCtx = new (window.AudioContext || window.webkitAudioContext)({`